packages/instrumentation-vertexai/tests/gemini.test.ts

```diff
@@ -27,7 +27,7 @@ import type * as vertexAiImport from "@google-cloud/vertexai";
 
 const memoryExporter = new InMemorySpanExporter();
 
-describe("Test Gemini GenerativeModel Instrumentation", () => {
+describe.skip("Test Gemini GenerativeModel Instrumentation", () => {
```

Comment:
@nirga  We should revert this `.skip`

Reply:
@5war00p yes we need to figure out how to mock these - see #58